tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -278,16 +270,25 @@ class WskRestAction
             ("", Base64.getEncoder.encodeToString(zip), artifactFile)
           }
           case ".js" => {
-            ("nodejs:default", FileUtils.readFileToString(new File(artifactFile)), artifactFile)
+            (
+              "nodejs:default",
+              FileUtils.readFileToString(new File(artifactFile), Charset.forName("UTF-8")),
```

Comment:
Replacing all of the charset getters with `StandardCharsets.UTF_8` would be nice.